cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,55 @@
+/// <reference types="cypress" />
+describe('Hermione Granger Account Flow', () => {
+  it('Should log in as Hermione Granger and perform deposit and withdrawal',
+    () => {
+      cy.visit(
+        'https://www.globalsqa.com/angularJs-protractor/BankingProject/#/login'
+      );
+      cy.contains('Customer Login', { timeout: 10000 })
+        .should('be.visible');
+
+      cy.contains('Customer Login').click();
+
+      cy.get('select').select('Hermoine Granger');
+
+      cy.contains('Login').should('be.visible').click();
+
+      cy.contains('Account Number').parent().should('contain', '1001');
+
+      cy.get('strong.ng-binding').should('contain', '5096');
+
+      cy.get('strong.ng-binding').contains('Dollar').should('be.visible');
+
+      cy.contains('Deposit').click();
+      cy.get('input[placeholder="amount"]').type('100');
+      cy.contains('Deposit').click();
+
+      cy.get('span.error.ng-binding[ng-show="message"]', { timeout: 10000 })
+        .should('not.have.class', 'ng-hide')
+        .and('contain', 'Deposit Successful');
```

Comment:
It's better to avoid complex selectors if possible. They make the test hard to read and maintain. Try to ask for adding data-* attributes to elements for testing purposes. If it's not possible, try to use simpler selectors.